clients/src/main/java/org/apache/kafka/common/record/CompressionType.java

```diff
@@ -156,7 +177,8 @@ else if (LZ4.name.equals(name))
     private static class LZ4Constructors {
         static final MethodHandle INPUT = findConstructor(
                 "org.apache.kafka.common.record.KafkaLZ4BlockInputStream",
```

Comment:
makes sense, as far as I know lz4 falls back to a pure-java version anyway, so it should be safe regardless (unless there are other platform issues that are unrelated to native code)